XenAdmin/TabPages/GeneralTabPage.cs

```diff
@@ -1943,6 +1943,12 @@ private void buttonViewLog_Click(object sender, EventArgs e)
             StartPutty( "env docker logs --tail=50 --follow --timestamps");
         }
 
+        private void panel2_Paint(object sender, PaintEventArgs e)
+        {   
+            //Force scrollbar to be repainted to avoid occasional pixel glitch.
+            panel2.AutoScroll = sections.Sum(s => s?.Parent.Height ?? s.Height) > panel2.ClientRectangle.Height;

```

Comment:
`s?.Parent.Height` is conflicting with `s.Height`. If s were null, then s.Height would be called, which would crash. I presume you probably meant to check for its parent's nullness, i.e. `s.Parent?.Height`

Reply:
Yes I did! Updated 👍